commands/help.js

```diff
@@ -28,16 +28,31 @@ module.exports = {
     let commandString = '```';
     let commandArray = [];
 
+/*    let authorRoles = [];
+    for (let role in message.author.roles){
+     authorRoles.push(role.get('name'));
+    }
+    console.log(authorRoles);*/
+
     // TODO: Display commands based on requireRoles
     for (let command in commands.commands) {
       let cmd = commands.commands[command];
       let info = '!' + command;
 
       // Skip commands that require roles for now
-      if (cmd.requireRoles) {
-        continue;
-      }
+      if (cmd.requireRoles){
+        let member = bot.guilds.first().members.get(message.author.id);
```

Comment:
In a recent branch I was handling it as a config/env var. @ThomasShaped advised against it at the time, but think mainly cause we had the above code everywhere else.

Should we move this logic to more of a `bot.guilds.find('name',appConfig.GUILD_NAME)` approach?
If so I can make the changes throughout on another PR.